File: python/pillow.py
class Solution(object):
    def passThePillow(self, n, time):
        """
        :type n: int
        :type time: int
        :rtype: int
        """
        mod = (time-1)%(n-1)
        div = (time-1)//(n-1)
        if div%2==0:
            return mod+2
        else:
            return n-(mod+1)
        

File: src/main/java/dailyChallege/Pillow.java
package dailyChallege;

public class Pillow {
	public int passThePillow(int n, int time) {
        int m = (time-1)%(n-1);
        if(((time-1)/(n-1))%2==0){
            return m+2;
        }else{
            return n-(m+1);
        }
    }

}
